MekHQ/data/universe/factions.xml

```diff
@@ -1101,6 +1101,8 @@ successor - unimplemented tag describing another faction code as the specified f
     <faction>
         <shortname>FC</shortname>
         <fullname>Federated Commonwealth</fullname>
+        <altNamesByYear year='3028'>Federated Commonwealth Alliance</altNamesByYear>
```

Comment:
This date would be too early. Per Fed Com Civil War - April 1st/3042 is when the AFFC was founded, but FedCom didn't come into being until the Death of Melissa on June 19th 3055. I would recommend using the 3042 date as that is when you'd see the military hiring Mercs.

Reply:
My understanding is that the Federated Commonwealth Alliance (as opposed to the Federated Commonwealth) came into being when the accords were signed in 3028. Then, in 3055 the alliance officially rebranded as the Federated Commonwealth in 3055, iirc, after things went poorly during the war of 3039.